src/components/Signup.jsx

```diff
@@ -54,6 +54,7 @@ const Signup = props => {
         </>
       );
       break;
+    default: signupFunction = null
```

Comment:
what is this?

Reply:
Working through the error messages coming from inspecting in the browser.  It said it expects a default case for switch statements

Reply:
okay cool